frontend/src/bundles/studio/components/player-controls/components/time-display/time-display.tsx

```diff
@@ -1,23 +1,24 @@
-import { format, secondsToMilliseconds } from 'date-fns';
+import { format } from 'date-fns';
 
 import { Flex, Text } from '~/bundles/common/components/components.js';
+import { useAppSelector } from '~/bundles/common/hooks/hooks.js';
 
-type Properties = {
-    currentTime: number;
-    duration: number;
-};
+const TimeDisplay: React.FC = () => {
+    const { elapsedTime, duration } = useAppSelector(({ studio }) => ({
+        elapsedTime: studio.player.elapsedTime,
+        duration: studio.player.duration,
+    }));
```

Comment:
```suggestion
    const { elapsedTime, duration } = useAppSelector(({ studio }) => studio.player);
```
We should use it like this because as far as i know if using your approach we receive a warning in a console. And the same for `useAppSelector` below

Reply:
Is this warning a problem? I just saw that we already use this approach with mapping object, for example in auth page, so I did the same for consistency

Reply:
In new PRs we are changing it, but I guess it is not critical